src/Microsoft.AspNet.Mvc.ViewFeatures/Controller.cs

```diff
@@ -784,9 +784,9 @@ public virtual FileStreamResult File(Stream fileStream, string contentType, stri
         /// </summary>
         /// <param name="fileName">The <see cref="Stream"/> with the contents of the file.</param>
         /// <param name="contentType">The Content-Type of the file.</param>
-        /// <returns>The created <see cref="FilePathResult"/> for the response.</returns>
+        /// <returns>The created <see cref="VirtualFilePathResult"/> for the response.</returns>
         [NonAction]
-        public virtual FilePathResult File(string fileName, string contentType)
+        public virtual VirtualFilePathResult File(string fileName, string contentType)
```

Comment:
Should the first argument be `filePath` \ `virtualPath`? Also cc @Eilon for design